internal/distro/distro.go

```diff
@@ -59,7 +59,7 @@ func DiskByPartUUIDDir() string { return diskByPartUUIDDir }
 func OEMDevicePath() string     { return fromEnv("OEM_DEVICE", oemDevicePath) }
 
 func KernelCmdlinePath() string { return kernelCmdlinePath }
-func SystemConfigDir() string   { return fromEnv("SYSTEM_CONFIG_DIR", systemConfigDir) }
+func SystemConfigDir() string   { return fromEnv("SYSTEM_CONFIG_DIR", "/home/core") }
```

Comment:
Why is this being changed?

Reply:
When running ignition and setting `-oem` as `packet`, ignition doesn't seem to take the exported env `SYSTEM_CONFIG_DIR`so I hard coded the path for the mean time.